processing-status-notifications-fa/src/main/kotlin/gov/cdc/ocio/exceptions/InvalidSchemaDefException.kt

```diff
@@ -0,0 +1,8 @@
+package gov.cdc.ocio.exceptions
+
+/**
+ * Intended use of this exception is for invalid schema definitions.
+ *
+ * @constructor
+ */
+class InvalidSchemaDefException(message: String): Exception(message)
```

Comment:
Do you need this?  I think it can be deleted.

Reply:
done. I added this and then i deleted all code around service bus but forgot to delete this one..